index.js

```diff
@@ -7,48 +7,47 @@ const schema = require('./src/schema/schema');
 const GitHubStrategy = require('passport-github2').Strategy;
 const passport = require('passport');
 const partials = require('express-partials');
-// const util = require('util');
 const session = require('express-session');
-const methodOverride = require('method-override');
-
+const cors = require('cors');
 const app = express();
+
 app.use(partials());
-app.use(methodOverride());
 
 const PORT = process.env.PORT || 8000;
 const GITHUB_CLIENT_ID = process.env.GITHUB_CLIENT_ID;
 const GITHUB_CLIENT_SECRET = process.env.GITHUB_CLIENT_SECRET;
+const SECRET = process.env.SECRET
 
-const allowCrossDomain = (req, res, next) => {
-  res.header('Access-Control-Allow-Credentials', true);
-  res.header('Access-Control-Allow-Origin', req.headers.origin);
-  res.header('Access-Control-Allow-Methods', 'GET,PUT,POST,DELETE');
-  res.header('Access-Control-Allow-Headers', 'X-Requested-With, X-HTTP-Method-Override, Content-Type, Accept');
-  if (req.method === 'OPTIONS') {
-    res.send(200);
-  } else {
-    next();
-  }
-};
+const LOCAL_CALLBACK_URL = process.env.LOCAL_CALLBACK_URL
+const NODE_ENV = process.env.NODE_ENV
 
+app.use(cors());
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: true }));
-app.use(allowCrossDomain);
 
 app.use(session({
-  secret: 'keyboard cat',
+  secret: process.env.SECRET,
   keys: [process.env.SESSION_KEY1, process.env.SESSION_KEY2],
   resave: false,
   saveUninitialized: false,
 }));
 
+
+
 app.use(passport.initialize());
 app.use(passport.session());
+passport.serializeUser((user, done) => {
```

Comment:
@Gwillison415 I think we need to figure out what this one is to at least write some comments so we can remember what it is for, thanks!